src/script/components/EphemeralTimer.tsx

```diff
@@ -0,0 +1,56 @@
+/*
+ * Wire
+ * Copyright (C) 2021 Wire Swiss GmbH
```

Comment:
```suggestion
 * Copyright (C) 2018 Wire Swiss GmbH
```
better to keep it as it's just a refactor with a new name and not something entirely new